test/integration/change-streams/change_stream.test.ts

```diff
@@ -1722,6 +1724,60 @@ describe('Change Streams', function () {
       });
     });
 
+    context('useBigInt64', () => {
+      const useBigInt64FalseTest = async function (options: ChangeStreamOptions) {
```

Comment:
```suggestion
      async function useBigInt64FalseTest(options: ChangeStreamOptions) {
```
or
```suggestion
      const useBigInt64FalseTest = async (options: ChangeStreamOptions) => {
```

Assigning an anonymous function to a variable is unusual - you'd typically just declare the function.
Alternatively, since we don't need to worry about binding `this`, we can just use an arrow function.